doc/modules/grid_search.rst

```diff
@@ -192,11 +192,46 @@ result in an error when using multiple metrics.
 See :ref:`sphx_glr_auto_examples_model_selection_plot_multi_metric_evaluation.py`
 for an example usage.
 
+.. _composite_grid_search:
+
 Composite estimators and parameter spaces
 -----------------------------------------
+`GridSearchCV` and `RandomizedSearchCV` allow searching over parameters of
+composite or nested estimators such as `Pipeline`, `ColumnTransformer`,
+`VotingClasssifier` or `CalibratedClassifierCV`
```

Comment:
link resolution here is kinda funny:
![links](https://user-images.githubusercontent.com/1663864/62618210-f22c0880-b913-11e9-85ed-d7f6aad8122e.png)

Reply:
why is ``VotingClassifier`` not resolved? Did I misspell it? Also: we can fix that later :P

Reply:
Pipeline should resolve to the Pipeline estimator doc, not to the user guide.

Reply:
Why? Isn't the user guide better?

Reply:
IMO, not when all the others links are estimators and when the link is actually a title section that often doesn't make sense in the context they're used in.

Reply:
maybe we should fix our title handles then? hm not sure, I don't have a strong opinion. I can use ``pipeline.Pipeline`` that should work, right? or use ``:ref:`pipeline.Pipeline```

Reply:
Yeah, I also think they all should link to comparable pages, i.e. the API page in this case. 

    :class:`ensemble.VotingClassifier`

should work for VotingClassifier.

Reply:
There was a typo ;)